src/main/webapp/testfile.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+
+
+<script src="SampleContent.java"></script>
+<!-- HTML Page setup for the use of having SampleContent.java as a source, purpose of UI -->
+

```

Comment:
Save this discussion for the pull request (where I am putting this comment; you can have inline conversations here).  It should not be in comments in the source code, because that's clutter, which is technical debt.